anabot/runtime/functions.py

```diff
@@ -266,6 +266,8 @@ def getselected(parent, visible=True):
     return getnodes(parent, visible=visible, selected=True)
 
 def log_screenshot(wait=None, progress_only=False):
+# Temporarily disabled because of broken screenshot functionality in dogtail
```

Comment:
Could this functionality be disabled just for wayland, so that we have screenshots on RHEL-9?

Also, could you add the RHEL issue number to the comments as well and fix the number it in the commit description? It should be RHEL-62420.

Also, s/RTT-5417/RTT-5714/ in the commit summary

Reply:
That's a very good point, I was so submerged in the RHEL-10 stuff that I forgot to address this separately. I fixed that and added logging messages informing that the screenshot wasn't taken due to broken screenshot functionality on Wayland. OTOH it can get quite spammy, but I still think it's better than no information at all.